### ISSUE TITLE
Enhancement: Add information about the different sources of bible

### ISSUE THREAD

**Issue by LAfricain** [crosswire/xiphos#984]
I would be great to have more information in module manager about the different repositories available, by instance, a popup window, or only a sentence which says:

> If you don't find a bible in your language, you can try to use an other repository as ebible.org...

**Comment by DavidHaslam** [comment]
See https://wiki.crosswire.org/Official_and_Affiliated_Module_Repositories

**Comment by LAfricain** [comment]
My idea is related to the problem with the duplicated module in Crosswire and ebible. More information could help the user who can't find his language, to search on another repository.

**Comment by karlkleinpaste** [comment]
i tend to think the problem of duplicated modules is best handled by removing duplicates. that is, the WEB modules really shouldn't be carried by Crosswire any more.
i usually return to the analogy of sword repos as publishers or bookstores. if you don't find what you want at one store, you go looking at another.

adding a line saying "look elsewhere" seems very kludgy. i don't know what else would suffice for what you seek, but that's really not it.

**Comment by LAfricain** [comment]
I agree with you about the duplicated modules. It's not only the web modules, but almost all chinese and Hindi. Maybe we need to seak about this on the mailing list. I spoke with the module team. For now they prefer to keep the two repo.
For now we have no information about the different  repo, I'm not sure a beginner will find easily. Do you think it's possible to update all the repo at once, as Bibletime, andbible and Bishop do?
If not, only a sentence saying: "you can find bible in different repositories in "Remote repositories"..." Something like that very easy.

**Comment by DavidHaslam** [comment]
A tool-tip over the legend for the MM **Add Standard** button might do the trick for new users (if such be feasible). 

To explain that it loads the connections for all the remote sources in the Master Repositories List. 

It’s not really a difficult concept.
